MayssaraOmar/assignment3/Word search/.cproject

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
It seems you accidentally added a number of files that are configuration files for your IDE.
I would suggest you delete those.